<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>450</width>
     <height>540</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="bot_page">
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>450</width>
       <height>600</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>430</width>
        <height>390</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>430</width>
        <height>390</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>430</width>
        <height>390</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollArea#scrollArea{border-color: rgb(170, 170, 255);
border-width:2px;
border-style:solid;
border-radius: 10px;}</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>426</width>
         <height>386</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>383</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTextEdit" name="chat_edit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>455</y>
        <width>371</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(170, 170, 255);
border-width:2px;
border-style:solid;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="send_chatbot_btn">
      <property name="geometry">
       <rect>
        <x>381</x>
        <y>455</y>
        <width>60</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>G마켓 산스 TTF Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 170, 255);
border-width:0px;
border-style:solid;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
</string>
      </property>
      <property name="text">
       <string>전송</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>G마켓 산스 TTF Bold</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(130, 130, 255);</string>
      </property>
      <property name="text">
       <string>챗봇과 채팅</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="user_page">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>450</width>
       <height>600</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>G마켓 산스 TTF Bold</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(130, 130, 255);</string>
      </property>
      <property name="text">
       <string>유저와 채팅</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>430</width>
        <height>390</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>430</width>
        <height>390</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>430</width>
        <height>390</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollArea#scrollArea_3{border-color: rgb(170, 170, 255);
border-width:1px;
border-style:solid;
border-radius: 10px;}</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>428</width>
         <height>388</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>385</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTextEdit" name="chat_edit_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>455</y>
        <width>371</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(170, 170, 255);
border-width:2px;
border-style:solid;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="all_send_btn">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>455</y>
        <width>60</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>G마켓 산스 TTF Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 170, 255);
border-width:0px;
border-style:solid;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
</string>
      </property>
      <property name="text">
       <string>전송</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="music_page">
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>450</width>
       <height>600</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>G마켓 산스 TTF Bold</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(130, 130, 255);</string>
      </property>
      <property name="text">
       <string>알람음 설정</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>400</width>
        <height>420</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width:2px;
border-style:solid;
</string>
      </property>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>221</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>G마켓 산스 TTF Medium</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(100, 100, 100);
border-width:0px;
border-style:solid;
</string>
       </property>
       <property name="text">
        <string>      군대 기상 나팔소리</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>77</y>
         <width>221</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>G마켓 산스 TTF Medium</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(100, 100, 100);
border-width:0px;
border-style:solid;
</string>
       </property>
       <property name="text">
        <string>      대피 사이렌 소리</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>130</y>
         <width>221</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>G마켓 산스 TTF Medium</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(100, 100, 100);
border-width:0px;
border-style:solid;
</string>
       </property>
       <property name="text">
        <string>      Good Morning</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>180</y>
         <width>221</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>G마켓 산스 TTF Medium</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(100, 100, 100);
border-width:0px;
border-style:solid;
</string>
       </property>
       <property name="text">
        <string>      Maple Story</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>490</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>G마켓 산스 TTF Medium</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(140, 140, 255);
color: rgb(255, 255, 255);
border-width:1px;
border-style:solid;
border-radius:10px;
</string>
      </property>
      <property name="text">
       <string>설정 완료</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="info_page">
    <widget class="QWidget" name="widget_8" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>450</width>
       <height>600</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>G마켓 산스 TTF Bold</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(130, 130, 255);</string>
      </property>
      <property name="text">
       <string>내정보 보기</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>48</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width:0px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../UI_img/프로필.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="nickname_lbl">
      <property name="geometry">
       <rect>
        <x>119</x>
        <y>111</y>
        <width>205</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>G마켓 산스 TTF Bold</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>이승신</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>145</y>
        <width>390</width>
        <height>380</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>390</width>
        <height>380</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>390</width>
        <height>380</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(140, 140, 140);
border-style:dashed;
border-width:2px;
border-radius:10px;</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>386</width>
         <height>376</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>386</width>
         <height>376</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>386</width>
         <height>376</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>373</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QPushButton" name="chatbot_btn">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>7</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0px;
border-style:solid;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../UI_img/챗봇2.png</normaloff>
      <normalon>../UI_img/챗봇.png</normalon>../UI_img/챗봇2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="userchat_btn">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>9</y>
      <width>70</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0px;
border-style:solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../UI_img/대화창.png</normaloff>
      <normalon>../UI_img/대화창2.png</normalon>../UI_img/대화창.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>65</width>
      <height>55</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="music_btn">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>6</y>
      <width>55</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0px;
border-style:solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../UI_img/음표.png</normaloff>
      <normalon>../UI_img/음표2.png</normalon>../UI_img/음표.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="myinfo_btn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>4</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0px;
border-style:solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../UI_img/내정보.png</normaloff>
      <normalon>../UI_img/내정보2.png</normalon>../UI_img/내정보.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
